docs/index.md

```diff
@@ -614,15 +614,11 @@ When prefetching with a `to_attr`, ensure that there are no collisions. Django d
 When prefetching *and* calling `select_related` on a field, Django may error, since the ORM does allow prefetching a selectable field, but not both at the same time.
 
 ### Changing `query` parameter name
-If you don't want to use the name `query` as your parameter, you can inherit `DynamicFieldsMixin` and change it as shown below
-
+If you don't want to use the name `query` as your parameter, you can change it with`QUERY_PARAM_NAME` on settings file e.g 
 ```py
-from django_restql.mixins import DynamicFieldsMixin
-class MyDynamicFieldMixin(DynamicFieldsMixin):
-    query_param_name = "your_favourite_name"
+QUERY_PARAM_NAME = "your_favourite_name"
```

Comment:
Usually it's best to namespace these parameters so that there might not be a collision. This could be `RESTQL_QUERY_PARAM_NAME`.

Reply:
I think we should use `rest_framework.settings.api_settings` instead of `django.conf.setting` so that all our settings are put under `REST_FRAMEWORK` ->  `RESTQL`
`

Reply:
Considering this requires DRF, I think that would work as well, as long as it's tied back to this package in an obvious way via the naming convention used.

Reply:
e.g
```py
REST_FRAMEWORK = {
    "RESTQL": {
         "QUERY_PARAM_NAME": "query",
    }
}
```
This will enable us to add other settings parameters easily in the future if we'll have more.

Reply:
Turns out DRF doesn't allow adding custom settings on its `api_setting` object, so I've created our own setting object `restql_setting`, with this configuration for **django-restql** is all namespaced inside a single Django setting named `RESTQL`.
For example a project's `settings.py` file might include something like this:
```py
RESTQL = {
    'QUERY_PARAM_NAME': 'query'
}
```